cmd/operator-sdk/bundle/validate.go

```diff
@@ -72,20 +75,38 @@ To build and validate an image:
 
 `,
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			if c.isToOutputJSON() {
```

Comment:
If we do not do it the result will be: (which is not we are looking for)

<img width="1446" alt="Screenshot 2020-05-22 at 14 09 37" src="https://user-images.githubusercontent.com/7708031/82735955-cbfa1c80-9d1d-11ea-8be5-0bb217424f85.png">

Instead of all converted to JSON.

Reply:
That is actually what we should see. Those logs are sent to stderr while JSON output is sent to stdout. When you run
```sh
$ operator-sdk bundle validate -o json-alpha1 | jq .
```
you will see no parse errors and logs will still print

Reply:
Oh. Cather the idea 👍